File: fragmentsjava/src/main/java/com/example/fragmentsjava/MainActivity.java
package com.example.fragmentsjava;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        loadFragment(new RedFragment());
    }

    public void loadFragment(Fragment fragment){
        FragmentManager manager =getSupportFragmentManager();
        FragmentTransaction trans=manager.beginTransaction();
        trans.replace(R.id.frameColor,fragment);
        trans.commit();

    }
}

File: app/src/main/java/com/example/lenovo/fragments/MainActivity.kt
package com.example.lenovo.fragments

import android.app.Fragment
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        loadFragment(Screen())
    }
    fun loadFragment(frag:android.support.v4.app.Fragment){
        val tns=supportFragmentManager.beginTransaction()
        tns.replace(R.id.frameLayout5,frag)
        tns.commit()
    }
}
